modules/rosa-sts-creating-cluster.adoc

```diff
@@ -41,11 +60,7 @@ $ rosa create cluster \
   --support-role-arn arn:aws:iam::${aws_account_id}:role/ManagedOpenShift-Support-Role \
```

Comment:
The role arns here depends on the output of last command `$ rosa create account-roles` once user input different prefix rather than `ManagedOpenShift` , then the command here will misleading users' creation

Reply:
Thank you for highlighting this. I have added the prefix to the `rosa create account-roles --version 4.8` command and have added call outs describing what is required if this is customised.